src/__tests__/medications/requests/NewMedicationRequest.test.tsx

```diff
@@ -184,20 +194,45 @@ describe('New Medication Request', () => {
       const medStatus = within(screen.getByTestId('statusSelect')).getByRole('combobox')
       const medicationIntent = within(screen.getByTestId('intentSelect')).getByRole('combobox')
       const medicationPriority = within(screen.getByTestId('prioritySelect')).getByRole('combobox')
+      const quantityValue = screen.getByLabelText(/quantityValue/)
+      const quantityUnit = screen.getByLabelText(/quantityUnit/)
+
+      userEvent.type(patient, 'Bruce')
+      await selectEvent.openMenu(patient)
+      await selectEvent.select(patient, /Bruce/)
 
-      userEvent.type(patient, 'Bruce Wayne')
       userEvent.type(medication, 'Ibuprofen')
       userEvent.type(medicationNotes, 'Be warned he is Batman')
-      selectEvent.create(medStatus, 'active')
-      selectEvent.create(medicationIntent, 'order')
-      selectEvent.create(medicationPriority, 'urgent')
+
+      await selectEvent.openMenu(medStatus)
+      await selectEvent.select(medStatus, /active/)
+
+      await selectEvent.openMenu(medicationIntent)
+      await selectEvent.select(medicationIntent, /order/)
+
+      await selectEvent.openMenu(medicationPriority)
+      await selectEvent.select(medicationPriority, /urgent/)
+
+      userEvent.type(quantityUnit, '200')
+      userEvent.type(quantityValue, 'mg')
 
       userEvent.click(
         screen.getByRole('button', {
           name: /medications\.requests\.new/i,
         }),
       )
-      expect(history.location.pathname).toEqual('/medications/new')
```

Comment:
I think this test was previously a false positive. The test description `should save the medication request and navigate to "/medications/:id"` but the submission was failing and the URL never progressed to the `/medications/:id`. I believe the test is now fixed and checks that on the submitted page that quantity unit and quantity value are present.